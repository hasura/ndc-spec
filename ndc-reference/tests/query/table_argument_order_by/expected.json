[
  {
    "rows": [
      {
        "id": {
          "value": 2
        }
      },
      {
        "id": {
          "value": 1
        }
      }
    ]
  }
]