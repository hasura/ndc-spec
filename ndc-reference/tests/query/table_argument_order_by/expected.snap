---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/table_argument_order_by/request.json
---
[
  {
    "rows": [
      {
        "id": 2
      },
      {
        "id": 1
      }
    ]
  }
]
